codice/ddf
# DDF-4134: Fixed export results error (while hidden)

#### What does this PR do?
Fixes export results component event listening while hidden on the page.
#### Who is reviewing it? 
@Corey-Collins 
@bellcc 
@brianfelix 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining

#### How should this be tested?
1. Search for results
2. `Add Visual` -> `Table`
3. Should see the 'export-as' component
4. Add filter so no results are visible
5. Should see no table or 'export-as' component and shouldn't be able to click near the search area or see error relating to export
#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4134](https://codice.atlassian.net/browse/DDF-4134)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
